lib/src/signed_video_tlv.c

```diff
@@ -282,13 +281,14 @@ encode_general(signed_video_t *self, uint8_t *data)
     write_byte(last_two_bytes, &data_ptr, (uint8_t)((timestamp)&0x000000ff), epb);
   }
 
-  // Write linked hash; hash_size bytes
+   // Write GOP hash; hash_size bytes
   for (size_t i = 0; i < self->sign_data->hash_size; i++) {
-    write_byte(last_two_bytes, &data_ptr, gop_info->linked_hashes[i], epb);
+    write_byte(last_two_bytes, &data_ptr, gop_info->computed_gop_hash[i], epb);
```

Comment:
Why did you change the order of linked hash and gop hash? The description above says linked hash is written before the gop hash.
You shouldn't have needed to change this file at all, since it was approved and committed separately.